backends/v2v/cfg/convert_destination.cfg

```diff
@@ -26,7 +26,7 @@ variants:
         sasl_pwd = v2v_tester_pwd
         remote_preprocess = yes
         remote_node_user = root
-        remote_node_password = redhat
+        remote_node_password = RHV_NODE_PASSWORD
```

Comment:
we could override with --vt-config as this might keep changing by time. any specific reason for this change ?

Reply:
redhat is not a specific variable,it is a really password.This patch the variable change to "RHV_NODE_PASSWORD" is a specific variable, we can replace it by CI dynamically in the future.

Reply:
cool then. ACK